src/au3audio/audioenginetypes.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Audacity: A Digital Audio Editor
+ */
+#pragma once
+
+#include "global/types/string.h"
+#include <optional>
+
+namespace au::audio {
+using EffectState = void*;
```

Comment:
Magic structure
Can we describe it? create our structure with important fields

Reply:
Was in fact just used as identifier. Just using `uintptr_t` is better indeed.